src/java/com/sarality/app/view/list/NonScrollingListViewInitializer.java

```diff
@@ -25,19 +25,30 @@
 public class NonScrollingListViewInitializer<T> extends BaseViewInitializer<LinearLayout, List<T>> {
 
   private final ListViewRowRenderer<T> rowRenderer;
+  private final ListViewRowRenderer<T> emptyViewRenderer;
 
   public NonScrollingListViewInitializer(FragmentActivity activity, LinearLayout listView,
-      ListViewRowRenderer<T> rowRenderer) {
+                                         ListViewRowRenderer<T> rowRenderer, ListViewRowRenderer<T> emptyViewRenderer) {
```

Comment:
Use the same Builder pattern as ListView Initializer

withEmptyView(View view, ViewRenderer<D> renderer, D data)